synapseclient/client.py

```diff
@@ -199,6 +199,7 @@ def __init__(self, repoEndpoint=None, authEndpoint=None, fileHandleEndpoint=None
         self.table_query_backoff = 1.1
         self.table_query_max_sleep = 20
         self.table_query_timeout = 600  # in seconds
+        self.multi_threaded = False  # if set to True, multi threaded download will be used for http and https URLs
```

Comment:
Do we need to expose this as settable from a command line option? I see how one using synapse as a Python library could set it but if someone was trying to use this feature from the command line is there a way to set it?